selectrum.el

```diff
@@ -278,6 +278,39 @@ This option is a workaround for 2 problems:
   wrapping."
   :type 'integer)
 
+(defcustom selectrum-candidate-transformations
```

Comment:
I think this would benefit from a more descriptive name. Maybe something like `selectrum-multiline-display-settings`?